Analytics/models/users.py

```diff
@@ -0,0 +1,151 @@
+from datetime import datetime
+from typing import NoReturn
+
+import bcrypt
+from sqlalchemy.exc import IntegrityError
+
+from db import db
+
+
+class Users(db.Model):
+    """
+    Database model for the layouts table used to persist widget layout data
+    :param id:          Primary key for user entry
+    :param fullname:    users full name
+    :param email:       users email addresas
+    :param admin:       true if the user is a admin, false if the user is not an admin
+    :param activated:   true if the account has been activated and false if it is not activated
+    :param timestamp:   time stamp of when the user was created.
+
+    :type id:           Integer
+    :type fullname:     String
+    :type email:        String
+    :type admin:        Boolean
+    :type activated:    Boolean
+    :type timestamp:    DateTime
+    """
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    fullname = db.Column(db.String(400), nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    password = db.Column(db.String(400), nullable=False)
+    admin = db.Column(db.Boolean)
+    activated = db.Column(db.Boolean)
+    timestamp = db.Column(db.DateTime)
+
+    def __init__(self, fullname, email, password, admin, activated, timestamp=None):
```

Comment:
Please use python3 typehiints, 
```suggestion
    def __init__(self, fullname: str, email: str, password: str, admin: bool, activated: bool, timestamp: datetime.datetime=None):
```